arkfilepicker/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="space.taran.arkfilepicker"/>
\ No newline at end of file
+    package="space.taran.arkfilepicker">
+    <uses-permission android:name="space.taran.arknavigator.ROOTS_READ"/>
```

Comment:
Do we need additional PR in Navigator or the apps?

Reply:
Yes, we need to set up a content provider that will provide roots, but it requires this PR